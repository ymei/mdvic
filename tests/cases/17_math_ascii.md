Inlined: $x^{2} + \alpha_{i}$ and $\frac{a}{b}$, $\sqrt{x}$, and $\sqrt[3]{y}$.

Display:
$$\frac{d}{dx}\sqrt{x} = \frac{1}{2\sqrt{x}}$$

