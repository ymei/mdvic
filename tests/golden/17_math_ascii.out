Inlined: x^2 + alpha_i and a/b, sqrt(x),
 and root[3](y).

Display: 
d/dxsqrt(x) = 1/2sqrt(x)


